src/commands/general/suggest.ts

```diff
@@ -0,0 +1,25 @@
+import Command, { SendFunction } from '../../util/BaseCommand';
+import SentinelClient from '../../client/SentinelClient';
+import { Message } from 'discord.js';
+import CommandArguments from '../../util/CommandArguments';
+import CommandError from '../../structures/CommandError';
+import Util from '../../util';
+
+export default class InviteCommand extends Command {
+	constructor(client: SentinelClient) {
+		super(client, {
+			aliases: [],
+			name: 'suggest',
+			dmAllowed: true,
+			description: 'Give the Sentinel Developers a suggestion!'
+		}, __filename);
+	}
+
+	async run(message: Message,  args: CommandArguments, send: SendFunction) {
+		const noSuggestion = new CommandError('PROVIDE_SUGGESTION');
+		if(!args[0]) throw noSuggestion;
```

Comment:
```suggestion
		if(!args[0]) throw new CommandError('PROVIDE_SUGGESTION');
```
theres no need for this to be a variable